.github/workflows/tests.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Install latest nightly Rust toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2021-03-18
+          toolchain: nightly-2022-05-09
```

Comment:
Is this the same as we use on centrifuge repo?

Reply:
Yeah, this is the same toolchain we're using in the centrifuge-chain repo :)

Reply:
Yeah, this is the same toolchain we're using in the centrifuge-chain repo :)